src/app/component/MissionGroup/MissionGroup.jsx

```diff
@@ -55,6 +55,14 @@ const MissionGroup = ({ callToAction, group, groupCallToAction, showViewRoute })
   };
   const onClickViewRouteButton = () => {
     console.log("View Routes");
+    missions.forEach((mission) => {
+      console.log(mission);
+      let pickUpLocation = mission.pickUpLocation.address;
+      let deliveryLocation = mission.deliveryLocation.address;
+      window.open(
```

Comment:
Is it possible to generate an anchor to indicate a page navigation other than a window.open? I dont think this will play well with accessibility and or just web dev practices. I think it's pretty uncommon to invoke window methods on the window object this directly

Reply:
I was told to use the server instead of connecting to Google Maps directly. Trying to figure that out right now.

Reply:
To be clear we wanna do this

1. Get the list of missions (and their addresses)
2. Send them to the server (API) and get back the optimum *ordering* of addresses
3. Take those addresses, in order, and send them to Google Maps directly as you are doing here.

Reply:
@jwu910 I understand that generally we want to have the user explicitly click the link but the thing is it takes time to generate the link - like 2-3 seconds - so we don't want to have that happen when this component renders, only after they click the button - at which point we need to pop open some sort of interstitial or something - and *then * navigate the use to the URL we just generated (which will open google maps).

In such a case would use of window.open be acceptable? Or.. would you still think it's  better in your opinion to generate ana actual anchor `<a href></a>` and then simulate the 'click' event on that anchor?

Reply:
Saw some changes to the code, didn't get a chance tor review yet, but wanted to comment on this.

As soon as the app knows about the stops, I think we want to generate that link. Even if its async, we should be able to run it as soon as the user is done with input.

The button can be disabled while we wait for teh async call, and then fill in the href src as soon as the results are back.

Thoughts?

Reply:
I like the idea of an async loading of that button - minimizes the time it will take to get to the route .. the only thing i worry about is that a volunteer in a hurry may become really frustrated by a button that  ... won't.. f**ng ...   click... 

I think the solution is to yes use an async thing to load the button but somehow do something that shows the user clearly that we are calculating the optimal route as to why the button is disabled... 

of course in MVP1 territory we may just have to store the address order in the group or something..

Reply:
This is react and we have the incremental dom!

We can easily just render a grey button that says loading route or something and then it can change colors when it’s ready.

If we’re having issues with the time it takes to come back from the API, I think we’d need to figure something out then. 

But with the features available to us with React, we have quite a bit of manipulative freedom

Reply:
OK cool.. thanks @jwu910 .. FWIW one of the reasons it takes a while is just that traveling salesperson problem is np complete so it may be that its actually not possible to make it go faster - that said I've suggested one thing to Vinh and he said he was gonna look into that. (I suggested that we hand his algorithm lat/lngs instead of asking his back end to do the geo-lookups)

Reply:
Implemented as Async using useEffect